src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -462,6 +463,24 @@ protected function getContextShopId()
      */
     protected function getErrorMessageForException(Exception $e, array $messages)
     {
+        if ($e instanceof ModuleEloquentException) {
+            return $e->getMessage();
+        }
+
+        $whitelist = [];
+        $this->get('prestashop.core.hook.dispatcher')->dispatchWithParameters('actionGetEloquentExceptionsFromModule', [
```

Comment:
I don't think that suffix `FromModule` is necessary as hooks are meant to be used only by modules.

Reply:
Indeed